### PR TITLE
Wrapping changelog in details for a collapsible view

### PR DESCRIPTION
**Pull request by Adorism**
## What
For all sources and destinations, this wraps the changelog in a "details" block. 

## User Impact
Preserves navigation functionality of having a changelog heading while minimizing the view of those changelogs which enhances browsability of the doc. 

## Can this PR be safely reverted and rolled back?

- [X] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by evantahler** [comment]
Tagging @timroes - does `<details>` work for the in-application version of the connector docs as well?

**Comment by timroes** [comment]
@evantahler Yes the `<details>` tag is working in app the same way.

File: docs/integrations/destinations/amazon-sqs.md [hunk 1]
```diff
@@ -117,10 +117,12 @@ The output SQS message would contain:
 <something>value</something>
 ```
 
-## CHANGELOG
-
+## Changelog
+<details>
+  <summary>Expand to review</summary>
```
[the remark this answers is not in the document]
Comment (Adorism) [reply docs/integrations/destinations/amazon-sqs.md:123]: Good catch. I've ensured all the tables are preceded and followed by a blank line.

**Review by evantahler — APPROVED** [review]
I spot checked a few connectors - they look good!

Vercel also says all page compile, so that's comforting.

**Comment by evantahler** [comment]
cc @alafanechere - we could add to the connector checks to not only have a `##changelog` section, but also that they are wrapped in a `<details>` block.

**Pull request merged**